test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -923,6 +923,8 @@ public Future<Void> shutdown(boolean graceful)
             
             error = parallelRun(error, executor, this::stopJmx);
 
+            DatabaseDescriptor.getCryptoProvider().uninstall();
```

Comment:
If we catch it then what do you want to do with that ... to log it? So we do all the stuff but crypto provider will remain to be installed. Then on the next test this will fail horribly (been there, done that) because it will use old crypto provider with new classloader of new Instance.

Reply:
As a fix I was thinking about the following `error = parallelRun(error, executor, () -> DatabaseDescriptor.getCryptoProvider().uninstall());`, but I think as the uninstall is called only in tests it should be safe from all sides.

Reply:
@Mmuzaf yes that works! wrapping it in parallelRun, good catch.